scripts/release/README.md

```diff
@@ -1,13 +1,13 @@
 # React Release Script
 
 At a high-level, the release script runs in 2 passes: **build** and **publish**.
-1. The **build** script does the heavy lifting (eg checking CI, running automated tests, building Rollup bundles) and then prints instructions for manual verification.
-1. The **publish** script then publishes the built artifacts to NPM and pushes to GitHub.
+1. The **build** script does the heavy lifting (e.g., checking CI, running automated tests, building Rollup bundles) and then prints instructions for manual verification.
+1. The **publish** script then publishes the built artifacts to npm and pushes to GitHub.
 
```

Comment:
I agree that this current wording ("The release script runs in 2 passes") is ambiguous. I see two situations:
1) If there is some sort of hidden release-script that uses build and publish, then we should clarify this.
2) If there is not, then we should write down that "At a high-level, the release-process uses 2 scripts: build and publish."
I think situation 2 is a lot more likely, and thus propose the above modification.

Reply:
Excactly, I was thinking of the word process as well (maybe without the '-' ?). However, what if they 'like' using the word 'release script'. For instance, what about the title of the readme then?

Reply:
The issue is further worsened by the fact that this is all inside a folder named "scripts." At this point I'm tempted to just add a release.js-file which simply runs **build** and **release** consecutively!

I think it's relatively fine for the readme's title to remain the same, even after our proposed changes. But we _could_ change it to "React Release Script**s**" instead.

Reply:
That's a really good idea although I myself am just unsure whether there is some sort of hidden yarn option which might do this already.

Reply:
Let's just try to change it to 'Script*s*' and await their response?

Reply:
Yes, let's.